src/styles/index.scss

```diff
@@ -1,3 +1,237 @@
+    height: 40px;
+    display: block;
+  }
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  text-transform: uppercase;
+
+  &__list {
+    list-style: none;
+    display: flex;
+    padding: 0;
+    margin: 0;
+    height: 100%;
+  }
+
+  &__item {
+    margin: 0 10px;
+    &:first-child {
+      margin-left: 0;
+    }
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    text-decoration: none;
+    color: $nav-links-color;
+    text-align: center;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    cursor: pointer;
+    height: 60px;
+    font-size: 12px;
+    transition-duration: 300ms;
+
+    &:hover {
+      color: $--blue-color;
+    }
+  }
+}
+
+.is-active {
+  position: relative;
+  color: $--blue-color;
+
+  &::after {
+    content: '';
+    display: block;
+    width: 100%;
+    height: 4px;
+    border-radius: 8px;
+    background-color: $--blue-color;
+    position: absolute;
+    bottom: 0;
+    left: 0;
+  }
+}
+
+.catalog {
+  gap: 46px 48px;
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  justify-content: center;
+}
+
+@media (min-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .catalog {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .catalog {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.product-card {
+  width: $card-width;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+  &__img {
+    width: 160px;
+    display: block;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-top: 0;
+    margin-bottom: 4px;
+    transition-duration: 300ms;
+
+    // &:hover {
+    //   color: #34568b;
+    // }
+  }
+  &__code {
+    margin-top: 0;
+    margin-bottom: 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary-color;
+  }
+  &__review {
+    margin-top: 0;
+    margin-bottom: 24px;
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    line-height: 16px;
+  }
+  &__reviews {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    margin: 0;
+  }
+  &__stars {
+    display: flex;
+    &-star {
+      height: 16px;
+      width: 16px;
+      background-image: url('../images/star.svg');
+      background-repeat: no-repeat;
+      background-position: center;
+      margin-right: 4px;
+    }
+    &-star:last-child {
+      margin-right: 0;
+    }
+    &-star:nth-child(-n + 4) {
+      background-image: url('../images/star-active.svg');
+    }
```

Comment:
This code will always make the first four stars active, regardless of the rating. You might want to consider a different approach here, perhaps using a loop or a function that takes the rating as a parameter and adds the active class accordingly.